src/chess/pgn.h

```diff
@@ -43,14 +45,31 @@ struct Opening {
   MoveList moves;
 };
 
+inline bool GzGetLine(gzFile file, std::string& line) {
+  bool flag = false;
+  char s[200];
+  line.clear();
+  while (gzgets(file, s, sizeof(s))) {
+    flag = true;
+    line += s;
+    auto r = line.find_last_of('\n');
```

Comment:
Any issue if someone tries to use a windows line ending pgn file on windows?

Reply:
I hope opening the file in "r" mode will take care of it - but a test would be nice.

Reply:
A quick test shows both line encodings seem to work fine under windows and linux.